RWS-Submission_Guidelines.md

```diff
@@ -1,34 +1,34 @@
-# First-Party Sets Submission Guidelines 
+# Related Website Sets Submission Guidelines 
 # Important Notice regarding Set Submissions 
-As Chrome prepares to [ship First-Party](https://groups.google.com/a/chromium.org/g/blink-dev/c/7_6JDIfE1as) Sets to General Availability (targeting a phased roll-out beginning Chrome 113),  we will shift from “testing” to “live” for First-Party Sets submissions. Please note key dates below as they relate to when your submissions will be applied to Stable behavior in Chrome:
+As Chrome prepares to [ship Related Website](https://groups.google.com/a/chromium.org/g/blink-dev/c/7_6JDIfE1as) Sets to General Availability (targeting a phased roll-out beginning Chrome 113),  we will shift from “testing” to “live” for Related Website Sets submissions. Please note key dates below as they relate to when your submissions will be applied to Stable behavior in Chrome:
```

Comment:
```suggestion
As Chrome prepares to [ship Related Website Sets](https://groups.google.com/a/chromium.org/g/blink-dev/c/7_6JDIfE1as) to General Availability (targeting a phased roll-out beginning Chrome 113),  we will shift from “testing” to “live” for Related Website Sets submissions. Please note key dates below as they relate to when your submissions will be applied to Stable behavior in Chrome:
```

Not strictly related to the renaming, but the existing grouping looks odd to me.